src/plugins/expression_reveal_image/public/expression_renderers/__stories__/reveal_image_renderer.stories.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { revealImageRenderer } from '../';
+import { elasticOutline, elasticLogo } from '../../../../presentation_util/public';
+
+// This is required only for stories, no need to put it to a bundle
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { Render } from '../../../../presentation_util/public/__stories__';
```

Comment:
@alexwizp, what do you think about such import?

Reply:
If you ask my opinion, I don't like cases where we need to turn off code validation. If I understand correctly, we will need to do similar imports from other plugins too.

What if we modify `.eslintrc.js` instead and allow imports from `/public/__stories__` like already works work `mock's` ?  

something like: 
![image](https://user-images.githubusercontent.com/20072247/122739385-96f72b00-d28b-11eb-955c-0d1d3806e931.png)

Reply:
I would reach out to kibana-operations to figure out the best way to handle this sharing of files between plugins but that are only intended for testing or storybook purposes.

Reply:
@elastic/kibana-operations please help us to find answer ^^^

Reply:
I chatted with ops.  Let's add a new zone here to allow importing from `__stories__` like Alexey suggests, but only from within a `__stories__` directory.

Reply:
Ok, thanks, @alexwizp  and @crob611  for your support.